code/modules/mob/living/simple_animal/heretic_monsters.dm

```diff
@@ -178,6 +178,7 @@
  */
 /mob/living/simple_animal/hostile/heretic_summon/armsy/Initialize(mapload, spawn_bodyparts = TRUE, worm_length = 6)
 	. = ..()
+	AddElement(/datum/element/blood_walk, /obj/effect/decal/cleanable/blood/tracks, target_dir_change = TRUE)
```

Comment:
you want to add this after spawn_bodyparts returns, as we only want the front to do gib trails

Reply:
gotcha